packages/jwt/src/test/test-utils.ts

```diff
@@ -0,0 +1,46 @@
+import type { jwtClaimsTestCases } from "../generated/jwt-claims-test-cases";
+import type { JWTPayload } from "../types";
+import { createAkashAddress } from "./seeders/akash-address.seeder";
+
+/**
+ * Replaces template values in JWT test cases with actual values
+ *
+ * Supports the following template patterns:
+ * - {{.Issuer}} - Replaced with a generated Akash address for the issuer
+ * - {{.Provider}} - Replaced with a generated Akash address for the provider
+ * - {{.IatCurr}} - Replaced with the current timestamp
+ * - {{.Iat24h}} - Replaced with a timestamp 24 hours in the past
+ * - {{.NbfCurr}} - Replaced with the current timestamp
+ * - {{.Nbf24h}} - Replaced with a timestamp 24 hours in the past
+ * - {{.Exp48h}} - Replaced with a timestamp 48 hours in the future
+ * @param testCase - The test case containing template values
+ * @returns The test case with template values replaced
+ */
+export function replaceTemplateValues(testCase: (typeof jwtClaimsTestCases)[0]) {
+  const now = Math.floor(Date.now() / 1000);
+  const issuer = createAkashAddress();
+  const provider = createAkashAddress();
+
+  const claims = { ...testCase.claims } as any;
+  if (claims.iss === "{{.Issuer}}") claims.iss = issuer;
```

Comment:
suggestion(non-blocking): if `claims` is a pure JSON then you can do it throught `JSON.parse(JSON.stringify(` combo. What will let you replace vars in any object inside claims without knowing its structure

```ts
const knownVars = {
  Issuer: issue,
  // ... other vars
};
const claims = JSON.parse(JSON.stringify(testCase.claims).replace(/"?\{\{\s*\.([\w_]+)\s*\}\}"?/g, (_, varName) => {
  return JSON.stringify(knownVars[varName]);
}))
```

Reply:
I wanted to point this out because if a Go developer decides to add formatting to the templates - for example, writing `{{ .Issuer }}` with extra spaces around var name - our current solution might break. This makes it somewhat fragile to minor changes.

Reply:
if you want to keep it as is, I'd recommend to write at least small utility to prevent this. Like `getTemplateVarName` and then

```ts
if (isTemplateVariable(claims.iss, "Issuer")) claims.iss = ...

function isTemplateVariable(rawValue: unknown, name: string): boolean {
  if (typeof rawValue !== 'string') return false;
  const value = rawValue.trim();
  return value.startsWith('{{') && value.endsWith('}}') && value.slice(2, -2) === name;
}
```